tests/test_experiment_rsmcompare.py

```diff
@@ -1,160 +1,29 @@
-    # experiment to itself with a custom list of sections
-    source = 'lr-self-compare-with-chosen-sections'
-    config_file = join(test_dir,
-                       'data',
-                       'experiments',
-                       source,
-                       'rsmcompare.json')
-    do_run_comparison(source, config_file)
-
-    html_report = join('test_outputs', source, 'lr_subgroups_vs_lr_subgroups_report.html')
-    yield check_report, html_report
-
-
-def test_run_experiment_lr_compare_with_custom_sections_and_custom_order():
-
-    # basic rsmcompare experiment comparing a LinearRegression
-    # experiment to itself with custom sections included and
-    # all sections in a custom order
-    source = 'lr-self-compare-with-custom-sections-and-custom-order'
-    config_file = join(test_dir,
-                       'data',
-                       'experiments',
-                       source,
-                       'rsmcompare.json')
-    do_run_comparison(source, config_file)
-
-    html_report = join('test_outputs', source, 'lr_subgroups_vs_lr_subgroups_report.html')
-    yield check_report, html_report
-
-
-def test_run_experiment_linearsvr_compare():
-
-    # basic rsmcompare experiment comparing an experiment
-    # which uses a SKLL model to itself
-    source = 'linearsvr-self-compare'
-    config_file = join(test_dir,
-                       'data',
-                       'experiments',
-                       source,
-                       'rsmcompare.json')
-    do_run_comparison(source, config_file)
-
-    html_report = join('test_outputs', source, 'LinearSVR_vs_LinearSVR_report.html')
-    yield check_report, html_report
-
-
-def test_run_experiment_lr_eval_compare():
-
-    # basic rsmcompare experiment comparing an rsmeval
-    # experiment to itself
-    source = 'lr-eval-self-compare'
-    config_file = join(test_dir,
-                       'data',
-                       'experiments',
-                       source,
-                       'rsmcompare.json')
-    do_run_comparison(source, config_file)
-
-    html_report = join('test_outputs', source, 'lr_eval_with_h2_vs_lr_eval_with_h2_report.html')
-    yield check_report, html_report
-
-
-def test_run_experiment_lr_eval_tool_compare():
-
-    # basic rsmcompare experiment comparing an rsmeval
-    # experiment to an rsmtool experiment
-    source = 'lr-eval-tool-compare'
-    config_file = join(test_dir,
-                       'data',
-                       'experiments',
-                       source,
-                       'rsmcompare.json')
-    do_run_comparison(source, config_file)
-
-    html_report = join('test_outputs', source, 'lr_with_h2_vs_lr_eval_with_h2_report.html')
-    yield check_report, html_report
+from parameterized import param, parameterized
+
+from rsmtool.test_utils import (check_run_comparison,
+                                do_run_comparison,
+                                rsmtool_test_dir)
+
+
+@parameterized([
```

Comment:
We lost `lr-self-compare-with-subgroups-and-h2`

Reply:
It's not in `rsmtool-refactor` either.

Reply:
That was added in the #117 branch that has not yet been merged. We'll address this when merging.